spyder/plugins/plots/widgets/tests/test_plots_widgets.py

```diff
@@ -77,6 +77,8 @@ def add_figures_to_browser(figbrowser, nfig, tmpdir, fmt='image/png'):
         figs.append(create_figure(figname))
         figbrowser._handle_new_figure(figs[-1], fmt)
 
+    figbrowser.thumbnails_sb.scene.setVerticalSpacing(6)
```

Comment:
Just wondering, why was this needed?

Reply:
I added it  to find the cause of the test failure in macos. Currently it was removed.